app/constants/app.ts

```diff
@@ -17,4 +17,8 @@ export const randomColor = (): string =>
 
 export enum TokenIds {
   MOB = 0,
+  // TODO for testing we are using 8192 instead of 1
+  // FIX-ME BEFORE MERGING INTO DEV, CHANGE THIS BACK TO 1
+  // MOBUSD = 1,
+  MOBUSD = 8192,
```

Comment:
Seems like this should maybe be changed to a 1? IDK, comments could be clearer... ;)

Reply:
We don't have any actual minted mUSD on testnet right now with token ID 1, so I'm using this token for development. Will make sure it's the right ID before merging this all into develop. I might even get the value from a call to the mobilecoin network